cnxdb/migrations/20170918131724_add_featured_books.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+
+
+def up(cursor):
+    cursor.execute("""\
+CREATE TABLE featured_books (
+    "uuid" UUID NOT NULL,
+    "major_version" INT,
+    "minor_version" INT
```

Comment:
let's add `tagid` (keeping the featured book tag here) so we don't lose what kind of feature it was. As well as a `fileid`, so we an link the cover art. This serves the use-case that UX and design will want different covers on the website, for design reasons (_e.g._ "let's make them all square"), than we may want as the canonical 'cover' for a book (in an epub or pdf).